src/components/HeaderBar.tsx

```diff
@@ -3,9 +3,8 @@ import { ReactComponent as DrawerHamburger } from '../assets/drawerHamburger.svg
 import { Link } from 'react-router-dom';
 
 export default function HeaderBar() {
-
 	return (
-		<div className="flex justify-between w-full p-8 shadow-headerbar mb-3">
+		<div className="flex justify-between w-full p-4 shadow-headerbar mb-3">
```

Comment:
Thanks! Also since u said it looks wrong on your monitor we may have to add chaning the paddings for xl screens?